test/README.md

```diff
@@ -277,6 +277,37 @@ $ cp jade/main/qemu/qemu_efuse.bin simulator/
 $ cd ..
 ```
 
+## Ledger emulator
```

Comment:
In fa5f38922274f2449162c34d74f92a1ab8e79450: This instruction only works on Linux, perhaps it'd be good to clarify it?